uach-retrofill.js

```diff
@@ -1,42 +1,50 @@
 /**
  * Copyright 2020 Google LLC
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
It's cool, I get it. You hate double quotes.

Reply:
It"s a compulsion.